package.json

```diff
@@ -86,8 +87,9 @@
 		"start": "npm run -s build",
 		"poststart": "run-p -s start-build-if-web start-build-if-desktop build-packages:watch",
 		"reformat-files": "./node_modules/.bin/prettier --ignore-path .eslintignore --write \"**/*.{js,jsx,json,ts,tsx}\"",
-		"start-build-fallback": "cross-env-shell BROWSERSLIST_ENV=defaults NODE_PATH=$NODE_PATH:server:client:. node NODE_ARGS=${NODE_ARGS:---max_old_space_size=8192} build/bundle.js",
-		"start-build-evergreen": "cross-env-shell BROWSERSLIST_ENV=evergreen NODE_PATH=$NODE_PATH:server:client:. node $NODE_ARGS --max_old_space_size=8192 build/bundle.js",
+		"start-build-do": "cross-env-shell node ${NODE_ARGS:---max_old_space_size=8192} build/bundle.js",
```

Comment:
i had no idea this syntax was possible `${NODE_ARGS:---max_old_space_size=8192}`

is that a cross-env-shell thing or a `npm scripts` thing?

Reply:
I think it may be missing `NODE_ARGS=` beforehand, judging by the other lines.

Reply:
It's a bash syntax for fancy parameter expansion and setting default values. See the "Default Values" section in https://devhints.io/bash#parameter-expansions

`cross-env-shell` does some transforms to the bash-like command line that make it compatible with Windows cmd.exe. But it won't understand the `${FOO:-val}` syntax and this will be one of many things broken in Windows build.

Reply:
I removed `cross-env-shell` from this particular script, as it doesn't set any environment variables.

Reply:
> I think it may be missing `NODE_ARGS=` beforehand, judging by the other lines.

`NODE_ARGS=` is needed only if the caller wants to add some Node.js command line options, or override the default value of `max_old_space_size`. Currently, the only place that does this is the `wp-desktop` build.